src/main/java/com/google/jenkins/plugins/k8sengine/KubeConfig.java

```diff
@@ -140,17 +140,18 @@ public KubeConfig build() {
    *
    * @param projectId The ID of the project the cluster resides in.
    * @param cluster The cluster data will be drawn from.
+   * @param credentialsId Service account credentials for GKE API access.
    * @return A {@link KubeConfig} from the specified {@link Cluster}.
    */
-  public static KubeConfig fromCluster(String projectId, Cluster cluster) {
+  public static KubeConfig fromCluster(String projectId, Cluster cluster, String credentialsId) {
```

Comment:
It appears credentialsId is now an accessToken. Need to update the name and javadoc to reflect.

Reply:
yikes, that must've been left here when git got in my way. Thanks

Reply:
done.